### DISCUSSION TITLE
A native exception occurred during launch

### DISCUSSION THREAD

**Discussion by dreamfly999** [microsoft/vscode-discussions#822] [Extension Development QnA] [answered]
when I  install the recent version, such as 1.82.2，I will get error when I connected to remote server by vscode. Just when I use the 1.52.1 is normal, but some plug-ins are not available. I don't know is it right ask for help here, but I really want to solve the problem.  
![image](https://github.com/microsoft/vscode-discussions/assets/34122505/78fbef76-2ccd-403e-89e1-cf6e27158c48)
error message:
A native exception occurred during launch (EBUSY: resource busy or locked, open '\.\pipe\conpty-5995336.540991683-in')

**Comment by gjsjohnmurray** [comment, marked as answer]
Please see https://github.com/microsoft/vscode/wiki/Submitting-Bugs-and-Suggestions for advice about reporting the issue. This discussion forum is currently focused on developers who are creating VS Code extensions.

**Comment by olguzzar** [comment]
Closing as this question is off-topic for this forum, which is focused on extension development.
